data/human/deep missions.txt

```diff
@@ -4042,10 +4045,10 @@ mission "Hjlod Remembers Windblain"
 		has "Hjlodsaved"
 	on offer
 		dialog
-			`Your inbox hums with a new message from <planet>:`
-			`	Dear <first>,`
+			`"Your inbox hums with a new message from <planet>:`
```

Comment:
```suggestion
			`Your inbox hums with a new message from <planet>:`
```

Yea, I didn't catch that loose one here, or the same swap in the next set like it.